File: Quarta/InputOutput/rettangolo/Rettangolo.java
package rettangolo;

import java.util.Scanner;
public class Rettangolo
{
    public static void main(String args[])
    {
        Scanner in = new Scanner(System.in);

        int base, altezza;
        int area, perimetro;
        
        System.out.print("Inserisci la base   : ");
        base = in.nextInt();
        System.out.print("Inserisci l'altezza : ");
        altezza = in.nextInt();

        perimetro = (base + altezza) * 2;
        area = base * altezza;

        System.out.println("il valore del perimetro e': " + perimetro);
        System.out.println("il valore dell'area e'    : " + area);
    }
}

File: Quarta/InputOutput/rettangolo/rettangolo.cs
using System;

namespace Rettangolo
{
    class Program
    {
        static void Main(string[] args)
        {
            double latoBase;
            double latoAltezza;
            double area;
            double perimetro;

            Console.WriteLine("Inserisci la base   : ");
            latoBase = Convert.ToDouble(Console.ReadLine());
            Console.WriteLine("Inserisci l'altezza : ");
            latoAltezza = Convert.ToDouble(Console.ReadLine());

            perimetro = (latoBase + latoAltezza) * 2;
            area = latoBase * latoAltezza;
            
            Console.WriteLine("il valore del perimetro e' :" + perimetro);
            Console.WriteLine("il valore dell'area e'     :" + area);
        }
    }
}